app/models/categorie.js

```diff
@@ -0,0 +1,8 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
+	name: attr('string'),
+	ID: attr('number')'
```

Comment:
no need to capitalize the id. Also, syntax error - there's an apostrophe instead of a comma at the end of the line. :)